src/components/Quiz.js

```diff
@@ -1,7 +1,8 @@
 import React, { useState } from 'react';
 import data from '../data/dummy.json';
-import { Button, Form, Container, Jumbotron } from 'react-bootstrap';
+import { Button, Form, Container } from 'react-bootstrap';
 import Question from './Question';
+import Result from "./Result";
```

Comment:
Prettier is still not auto-formatting since there are `"` here. Can you make sure Prettier is auto-formatting?

Reply:
I will keep looking how to fix that.